app/models.py

```diff
@@ -419,6 +419,18 @@ def get_tile_json_data(self):
             }
         }
 
+    def get_textured_mesh_archive(self):
+        archive_path = self.assets_path("odm_texturing.zip")
+        textured_mesh_directory = self.assets_path("odm_texturing")
```

Comment:
I don't know what self.assets_path refers to, but you might not want to hard-code the name here